### PR TITLE
Add filter by referrer feature (REFERRERS_FILTER)

### PR DESCRIPTION
**Pull request by abulte**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -6,3 +6,7 @@
 FORCE_SSL = False
 # In bytes, cf `sniff_limit` https://agate.readthedocs.io/en/1.6.1/api/table.html#agate.Table.from_csv
 CSV_SNIFF_LIMIT = 4096
+# Set this to an array of hosts to filter calls by referer
+# It will also match subdomains
+# e.g. REFERERS_FILTER = ['data.gouv.fr'] will match 'demo.data.gouv.fr'
```
Comment (noirbizarre) [inline config.py]: `REFERERS` or `REFERRERS` ?

File: config.py [hunk 2]
```diff
@@ -9,3 +9,7 @@
 # In bytes, csvapi will stop downloading files if they reach this size
 # Default to 100 Mo
 MAX_FILE_SIZE = 1024 * 1024 * 100
+# Set this to an array of hosts to filter calls by referer
+# It will also match subdomains
+# e.g. REFERERS_FILTER = ['data.gouv.fr'] will match 'demo.data.gouv.fr'
```
Comment (noirbizarre) [inline config.py]: REFER**R**ERS_FILTER

File: config.py [hunk 3]
```diff
@@ -9,3 +9,7 @@
 # In bytes, csvapi will stop downloading files if they reach this size
 # Default to 100 Mo
 MAX_FILE_SIZE = 1024 * 1024 * 100
+# Set this to an array of hosts to filter calls by referer
```
Comment (noirbizarre) [inline config.py]: filter out

Comment (noirbizarre) [reply config.py]: Maybe add a note about the fact that a 403 will be returned

File: config.py (hunk 2 above)
Comment (abulte) [reply config.py]: yep, I changed my mind in the mean time ;-)

**Pull request merged**